test/integration/serviceaccount/service_account_test.go

```diff
@@ -371,18 +371,26 @@ func TestLegacyServiceAccountTokenTracking(t *testing.T) {
 			myConfig.BearerToken = string(test.secretTokenData)
 			roClient := clientset.NewForConfigOrDie(&myConfig)
 			dateBefore := time.Now().UTC().Format(dateFormat)
-			go func() {
-				doServiceAccountAPIRequests(t, roClient, myns, true, true, false)
-			}()
-			doServiceAccountAPIRequests(t, roClient, myns, true, true, false)
+
+			var wg sync.WaitGroup
+			concurrency := 5
+			for i := 0; i < concurrency; i++ {
+				wg.Add(1)
+				go func() {
+					doServiceAccountAPIRequests(t, roClient, myns, true, true, false)
+					wg.Done()
```

Comment:
```suggestion
                                          defer wg.Done()
					doServiceAccountAPIRequests(t, roClient, myns, true, true, false)
```

Reply:
one line to defer? probably not.

Reply:
is the common pattern, if in the future you add more lines, this is always covered ... but is only nit, don't worry